dummyserver/testcase.py

```diff
@@ -292,6 +294,26 @@ class IPv6HTTPDummyProxyTestCase(HTTPDummyProxyTestCase):
     proxy_host_alt = "127.0.0.1"
 
 
+class HypercornDummyServerTestCase:
+    host: str = "localhost"
+    port: str = "8080"
+    base_url: typing.ClassVar[str]
+
+    _stack: contextlib.ExitStack
+
+    @classmethod
+    def setup_class(cls) -> None:
+        with contextlib.ExitStack() as stack:
```

Comment:
It would have been easy to use a pytest fixture with the `run_hypercorn_in_thread` context manager, but I wanted to mirror the existing `TestCase` convention which makes it very easy to attach class variables like `base_url`.